pandas/io/parsers.py

```diff
@@ -1765,6 +1769,23 @@ def _cast_types(self, values, cast_type, column):
                 cats, cats.get_indexer(values), cast_type,
                 true_values=self.true_values)
 
+        # use the EA's implementation of casting
+        elif is_extension_array_dtype(cast_type):
+            try:
+                array_type = cast_type.construct_array_type()
+            except AttributeError:
+                cast_type = pandas_dtype(cast_type)
```

Comment:
why are you catching AttributeError here? this should *always* exist, it is a required feature of EAs (if it doesn't then we need to fix that).

What is this failing on ?

Reply:
Don't need this. It was before it was in this `if` block, just didn't remove it.